### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by drwill-ms**
Notes inline.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,51 +1,55 @@
 # Microsoft Azure IoT SDK for .NET
 
 ### Help us help you with our IoT SDKs!
-We are running a survey through August 2020 to learn more about your IoT projects and support needs. Our team will use this information to help shape the future of our IoT SDKs, and, if you choose to provide your contact information, we'll include you in our circle of advisors for early feedback. Consider spending ~5 minutes completing **[this survey](https://aka.ms/iotsdksurvey)**-- we'd love to hear from you!
+
+We are running a survey through August 2020 to learn more about your IoT projects and support needs.
```
Comment (drwill-ms) [inline readme.md:5]: Making newlines after each sentence makes it easier to read the raw text, but doesn't change the rendered format.

File: readme.md [hunk 2]
```diff
@@ -1,51 +1,55 @@
...
-| ---                                                                   | ---    |
-| [Master](https://github.com/Azure/azure-iot-sdk-csharp/tree/master)   | [![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/csharp/CSharp%20Prod%20-%20West%20Central%20US?branchName=master)](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_build/latest?definitionId=44&repositoryFilter=9&branchName=master)|
-| [Preview](https://github.com/Azure/azure-iot-sdk-csharp/tree/preview) | [![Build Status](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_apis/build/status/csharp/CSharp%20Canary%20-%20Central%20US%20EUAP?branchName=preview)](https://azure-iot-sdks.visualstudio.com/azure-iot-sdks/_build/latest?definitionId=402&repositoryFilter=9&branchName=preview)|
+| Service Environment                                                   | Status                                                                                                                                                                                                                                                                                        |
```
Comment (drwill-ms) [inline readme.md:22]: Lots of table alignment fixes.

File: readme.md [hunk 3]
```diff
@@ -1,51 +1,55 @@
...
+| Microsoft.Azure.Devices.DigitalTwin.Client            | N/A                                                       | [![NuGet][pnp-device-prerelease]][pnp-device-nuget]           |
+| Microsoft.Azure.Devices.DigitalTwin.Service           | N/A                                                       | [![NuGet][pnp-service-prerelease]][pnp-service-nuget]         |
 
 The API reference documentation for .NET SDK is [here][dotnet-api-reference].
 
 To find SDKs in other languages for Azure IoT, please refer to the [azure-iot-sdks][azure-iot-sdks] repository.
 For IoT Hub Management SDK in .NET, please visit [azure-sdk-for-net](https://github.com/Azure/azure-sdk-for-net) repository
 
-## Need Support?
+## Need support?
```
Comment (drwill-ms) [inline readme.md:48]: Sentence case for headers.

File: readme.md [hunk 4]
```diff
@@ -54,19 +58,20 @@ Visit [Azure IoT Dev Center][iot-dev-center] to learn more about developing appl
 
 ## Samples
 
-Samples are available at [Azure IoT Samples for C#](https://github.com/Azure-Samples/azure-iot-samples-csharp)
+Most of our samples are available at [Azure IoT Samples for C#](https://github.com/Azure-Samples/azure-iot-samples-csharp).
+
+If you are looking for a good device sample to get started with, please see the [device reconnection sample](https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/device/DeviceReconnectionSample).
```
Comment (drwill-ms) [inline readme.md:63]: I feel we should highlight the device reconnection sample more.

File: readme.md [hunk 5]
```diff
@@ -54,19 +58,20 @@ Visit [Azure IoT Dev Center][iot-dev-center] to learn more about developing appl
...
 If you would like to build or change the SDK source code, please follow the [devguide](doc/devguide.md).
 
 ## OS platforms and hardware compatibility
 
-> .NET MicroFramework will not be supported in future versions of the SDK.
-
 > .NET Standard 1.3 (IoT Hub SDKs only) is last supported in the [2020-02-27](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/2020-2-27) and in the [2020-1-31 LTS](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/lts_2020-1-31) releases.
 
-The IoT Hub device SDK for .NET can be used with a broad range of OS platforms and devices.
+The IoT Hub device SDK for .NET can be used with a broad range of OS platforms and devices, mainly modern versions Windows and Linux.
```
Comment (drwill-ms) [inline readme.md:74]: The former statement was too vague. Even this is too vague, but it is a step in the right direction.

File: readme.md [hunk 6]
```diff
@@ -185,16 +188,16 @@ Below is a table showing the mapping of the LTS branches to the packages release
...
 
-Microsoft collects performance and usage information which may be used to provide and improve Microsoft products and services and enhance your experience.  To learn more, review the [privacy statement](https://go.microsoft.com/fwlink/?LinkId=521839&clcid=0x409).  
+Microsoft collects performance and usage information which may be used to provide and improve Microsoft products and services and enhance your experience.
+To learn more, review the [privacy statement](https://go.microsoft.com/fwlink/?LinkId=521839&clcid=0x409).
 
 [iot-hub-documentation]: https://docs.microsoft.com/en-us/azure/iot-hub/
 [iot-dev-center]: http://azure.com/iotdev
 [azure-iot-sdks]: https://github.com/azure/azure-iot-sdks
 [dotnet-api-reference]: https://docs.microsoft.com/en-us/dotnet/api/overview/azure/iot/client?view=azure-dotnet
-[devbox-setup]: ./doc/devbox_setup.md
```
Comment (drwill-ms) [inline readme.md:196]: Unreferenced links

File: readme.md (hunk 1 above)
Comment (timtay-microsoft) [inline readme.md:5]: Is August 2020 already over? Should we delete this?

File: readme.md [hunk 7]
```diff
@@ -152,14 +157,12 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 
 > Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
 
-# Long Term Support
+# Long-Term Support (LTS)
 
 The project offers a Long Term Support (LTS) version to allow users that do not need the latest features to be shielded from unwanted changes.
 
-As of August 2020, the .NET SDK is shifting to a revised Long-Term Support strategy. The primary motivations for this change are to extend the support period and decrease the churn on LTS releases, while still maintaining a strategy that offers customers choice between new features and stability.
```
Comment (abhipsaMisra) [inline readme.md:159]: This text was provided by Elena to highlight our LTS strategy, and the motivation behind supporting it for 3 years. I feel we should retain this for one LTS release cycle.

File: readme.md [hunk 8]
```diff
@@ -152,14 +157,12 @@ This repository contains [provisioning service client SDK](./provisioning/servic
...
+# Long-Term Support (LTS)
 
 The project offers a Long Term Support (LTS) version to allow users that do not need the latest features to be shielded from unwanted changes.
 
-As of August 2020, the .NET SDK is shifting to a revised Long-Term Support strategy. The primary motivations for this change are to extend the support period and decrease the churn on LTS releases, while still maintaining a strategy that offers customers choice between new features and stability. 
-
-We now will be releasing a new LTS branch yearly, and each LTS release will be supported for 3 years - 1 year of active maintenance with bugfixes, and 2 years of extended support for security fixes.
-
+A new LTS version will be created every 6 months.
+The lifetime of an LTS branch is currently planned for one year.
```
Comment (abhipsaMisra) [inline readme.md:165]: We have 1 year of bug fix + 2 years of security fixes - we should highlight that.

**Review by abhipsaMisra — APPROVED** [review]
One comment about retaining the existing statement about LTS release strategy, looks good otherwise!

File: readme.md (hunk 1 above)
Comment (abhipsaMisra) [reply readme.md:5]: @sharmasejal Is this still on?

**Review by vinagesh — APPROVED** [review]

File: readme.md (hunk 7 above)
Comment (drwill-ms) [reply readme.md:159]: This change was unintnentional. I will restore it.

File: readme.md (hunk 1 above)
Comment (sharmasejal) [reply readme.md:5]: Yes, this is on. I will keep posted once this is done.

Comment (sharmasejal) [reply readme.md:5]: This is closing next Friday, 9/25

**Review by azabbasi — APPROVED** [review]

**Review by timtay-microsoft — APPROVED** [review]

**Pull request merged**